lib/backend.js

```diff
@@ -43,43 +46,20 @@ function loadDynamoEndpoint(env, dynamoConfig) {
  * @param AWS - the AWS SDK object
  * @returns {{endpoint: string, sslEnabled: boolean, region: string, accessKeyId: string}}
  */
-function loadDynamoConfig(env, AWS) {
+function loadDynamoConfig(env) {
   const dynamoConfig = {
     endpoint: 'http://localhost:8000',
     sslEnabled: false,
-    region: 'us-east-1',
-    accessKeyId: 'key',
-    secretAccessKey: env.AWS_SECRET_ACCESS_KEY || 'secret'
+    region: 'us-east-1'
   }
 
   loadDynamoEndpoint(env, dynamoConfig)
-
-  if (AWS.config) {
-    if (AWS.config.region !== undefined) {
-      dynamoConfig.region = AWS.config.region
-    }
-
-    if (AWS.config.credentials) {
-      if (AWS.config.credentials.accessKeyId !== undefined) {
-        dynamoConfig.accessKeyId = AWS.config.credentials.accessKeyId
-      }
-    }
-  }
-
-  if (env.AWS_REGION) {
-    dynamoConfig.region = env.AWS_REGION
-  }
-
-  if (env.AWS_ACCESS_KEY_ID) {
-    dynamoConfig.accessKeyId = env.AWS_ACCESS_KEY_ID
-  }
-
   return dynamoConfig
 }
 
-const createAwsConfig = (AWS) => {
+const createAwsConfig = () => {
```

Comment:
I'm running dynamodb on port 8042 locally. Running

```sh
DYNAMO_ENDPOINT=http://localhost:8042 nr dev --port 8002
```

on this branch and opening http://0.0.0.0:8002 results in error:

> CredentialsProviderError: Could not load credentials from any providers

I suppose because the default access key and secret are no longer specified?

Reply:
sorry it took me a while to respond.

Yeah it is failing when there are no credentials configured locally.
By default it would follow the sdk and search for creds